addons/mail/static/src/models/composer/composer.js

```diff
@@ -518,6 +605,12 @@ function factory(dependencies) {
         discussAsReplying: one2one('mail.discuss', {
             inverse: 'replyingToMessageOriginThreadComposer',
         }),
+        elementModel: attr({
+           default: "",
+        }),
+        extraListName: attr({
+           default: "",
+        }),
```

Comment:
will need to add "suggestion" in these names to contextualize them + need comments to explain what they do